### PR TITLE
spellcheck

### PR DESCRIPTION
**Pull request by decanus**

### CODE REVIEW

File: docs/stable/6-payloads.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ title: 6/PAYLOADS
 
 ## Abstract
 
-This specifications decribes how the payload of each message in Status looks
-like. It is primarly centered around chat and chat-related use cases.
+This specifications describes how the payload of each message in Status looks
+like. It is primarily centered around chat and chat-related use cases.
 
 The payloads aims be flexible enough to support messaging but also cases
```
Comment (Samyoul) [inline docs/stable/6-payloads.md]: I spotted this on while doing #114, if there are any other grammatical issues that you didn't cover already I'll address them here.

```suggestion
The payloads aim to be flexible enough to support messaging but also cases
```

**Review by corpetty — APPROVED** [review]
LGTM

File: docs/stable/6-payloads.md (hunk 1 above)
Comment (decanus) [reply docs/stable/6-payloads.md]: @Samyoul let's not do a all grammar issues here. I don't want this to get too large

**Comment by Samyoul** [comment]
What do you think about changing https://github.com/status-im/specs/pull/119/files#diff-53565848b9b7dcb282e708743790874aR338

to

```md
   * `symKeyID` - `String`: an ID of a symmetric key used to authenticate with the `Mailserver`, derived from Mailserver password.
```

**Pull request merged**